easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.2-foss-2016a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,27 @@
+name = 'SuiteSparse'
+version = '4.5.2'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
+
+preconfigopts = 'MY_METIS_LIB="$EBROOTMETIS/lib/libmetis.so" ' 
+preconfigopts += 'LAPACK="$LIBLAPACK" '
```

Comment:
SuiteSparse easyconfig was taked from PR #2766 and I just add MY_METIS_LIB. If I don't include 'LAPACK="$LIBLAPACK" one error occur in building step because lapack.
But now I see that a rumtime error occur in Yade (problem with cholmod ) because I include SuiteSparse so I'm going to remove this dependency and disable the fluid coupling module in Yade that is unnecessary for now